envi/archs/arm/__init__.py

```diff
@@ -56,6 +56,7 @@ class ThumbModule(envi.ArchitectureModule):
     archVersion = ('ARMv7A')
 
     def __init__(self, name='ARMv7A'):
+        self.archVersion = name
```

Comment:
i'm not sure this will do what you think it will do :)  but we'll let it go for now.